content/contributing/adding-content/adding-articles-to-rkvst-docs/index.md

```diff
@@ -22,9 +22,9 @@ toc: true
     ```
     Here you can see we have specified the `contributing` area, the `adding content` section and the name of our new article with a non-underscored `index.md` to indicate this is an article
     {{< note >}}
```

Comment:
For some reason the indentation of this Note block breaks how it is displayed in my browser. Deleting some of the indentation fixes it.